corehq/apps/export/views.py

```diff
@@ -766,6 +766,17 @@ def _process_filters_and_specs(self, in_data):
 
         return export_filter, export_specs
 
+    def get_export_documents_count(self, in_data):
```

Comment:
Exports can get big and slow. I think it'd be worth changing this code to `has_export_documents` and returning true as soon as possible rather than always getting all filters/specs and counting all the documents. @esoergel any thoughts?

Reply:
@orangejenny this sounds reasonable. I'll change this after Ethan confirmation. This should be a quick one.

Reply:
I agree with Jenny that this should just check for any documents at all, then return as early as possible.  It looks like `get_export_documents` actually fetches all the documents in the export, which is very costly.  Instead, this should check [`get_export_size`](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/export/export.py#L351-L352).

Even better would be if this can go in the export task itself and inspect the results returned to see if there are any documents.  That may not be simple though, I haven't investigated.